### ISSUE TITLE
Maven Download Support

### ISSUE THREAD

**Issue by Pkmmte** [lewisjdeane/L-Navigation-Drawer#1]
Just something minor. it'd be nice to be able to download this library via Maven rather than Dropbox. It just makes everything a lot easier. Thanks and great library! :+1:

**Comment by lewisjdeane** [comment]
Yeah i've been wanting to do this but I haven't really known how to do it, do you have any advice/experience regarding Maven? And thanks i'm glad you like it!

**Comment by Pkmmte** [comment]
I'm rather new-ish to Maven but I can get around it. Feel free to message me via Google Hangouts or email (admin[at]pkmmte[dot]com) if you want me to guide you through the entire process. :3

**Comment by lewisjdeane** [comment]
Okay that would be great I'll drop you a hangouts in a bit, thanks!

**Comment by outadoc** [comment]
I hope that gets fixed, it would be a whole lot easier to install! :3

**Comment by nanoxd** [comment]
Has this been moved to Maven yet?

**Comment by jaydp17** [comment]
any updates ?
